### ISSUE TITLE
[Enhancement] Ability to imbue negative potion effect levels

### ISSUE THREAD

**Issue by Collserra** [DieReicheErethons/Brewery#163]
Some potion effects, such as "Jump" or "Resistance" actually have a different effect if given to a player with a negative level.

For example, giving the player jump boost with a level of "-7" will make it impossible for the player to jump. This can be useful in some situations... For example, making an unrealistic brew that causes the player to freeze momentarily, giving slowness 7 (no movement) and jump boost -5 (no jumping), along with blindness for a nice "frozen" effect. I have made potions with this in the past, but thought that using brewery would be the perfect way to allow players to brew this.

tl;dr

Allow the use of negative potion levels (Jump Boost -5 (no jumping)) in the recipes config for more effects.
